### DISCUSSION TITLE
Use diec via macOS installer

### DISCUSSION THREAD

**Discussion by shellcromancer** [horsicq/Detect-It-Easy#119] [Q&A] [answered]
Hello! Thanks for building this project. :) 

Is there a way to use the CLI tool via the macOS installer? I didn't see the executable in the .app bundle so I wasn't sure if it was included.

**Comment by horsicq** [comment, marked as answer]
Hello! Thanks a lot for your report. I will include diec in the next version of DiE.

**Reply by ITAYC0HEN** [reply]
Looking forward to it :)
